doc/dev-reference/conf.py.in

```diff
@@ -43,7 +43,7 @@ source_suffix = '.rst'
 master_doc = 'index'
 
 # General information about the project.
-project = u'Hyperscan'
+project = u'Vectorscan'
```

Comment:
such changes need checking, for a start, we cannot change the project name like that, and we need to add the relevant copyright for our company VectorCamp for the years 2020-2024. In general references to the project can be changed, but when it comes to copyright mentions, I will need to look over them. In fact, I would put everywhere possible that this project is a fork of Hyperscan, the original Intel project.

Reply:
INAL, but I don't understand the part about not changing the project name. It's already been changed, no? So it probably needs to stop calling itself 'hyperscan' (is that an intel trademark?)

I didn't want to start going down this rabbit hole, but the man pages were coming out as hyperscan.7 and saying hyperscan everywhere rather than vectorscan. 

But I'm fine with all the rest of the review comments, it's just this one I don't fully understand. 


PS: I also should have updated the pkgconfig description, although for API/ABI compatibility, i'm assuming that has to remain as libs.pc.

Reply:
Also, I probably won't update this till tue, since its a long weekend.

Reply:
I should be clearer, I meant the copyright below. For example, Hyperscan is Intel copyright (years up to current), Vectorscan is Intel copyright (years up to 2020, that's when we forked) AND VectorCamp PC 2020-2024 and in some files the respective copyrights of some contributors for some patches, eg. Arm Ltd. These statements have to be fixed.

Reply:
But also IANAL, need to consult some people on that to be totally correct.